src/components/Loader/PeopleTable.tsx

```diff
@@ -0,0 +1,69 @@
+// #region imports
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import React from 'react';
+// #endregion
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const { sex, born, died, motherName, fatherName } = person;
```

Comment:
The destructuring assignment here assumes that `motherName` and `fatherName` are properties of the `person` object. Ensure that these properties exist in the `Person` type definition.